packages/devtools_app/lib/src/performance/flutter/performance_screen.dart

```diff
@@ -131,37 +153,86 @@ class _PerformanceScreenBodyState extends State<PerformanceScreenBody>
         ),
       ],
     );
+
+    // We put these two items in a stack because the screen's UI needs to be
```

Comment:
we need the screen to be built because there are listeners that are initialized when the screen widget is built. I ran into this issue in the Timeline screen and landed on the stack implementation because if the screen is not built ahead of time, the listeners do not respond to events that happen while the data is being processed.